sonar-project.properties

```diff
@@ -0,0 +1,6 @@
+sonar.projectKey=bears
+sonar.projectBaseDir=.
+sonar.javascript.lcov.reportPaths=coverage/lcov.info
+sonar.language=js
+sonar.tests=test
+sonar.sources=assets,components,content,layouts,middleware,mixins,pages,plugins,report,services,static,store
```

Comment:
This is where the list of directories are listed.

SonarQube can be given a path to start scanning; however, if we say "." (i.e., the project root), then it'll scan test code and include the results in with the actual code, so we give it paths to the source and to the tests.  Unfortunately, we can't say the source code is "not test".  Also, we want to avoid scanning `node_modules` because ugh.

Reply:
So, @flexyjesse @fpigeonjr @lgarceau768 please help me refine which directories should (or should not) be scanned.  Also, files / directories matching patterns from .gitignore are ignored (see `sonar.scm.exclusions.disabled`)

Reply:
https://docs.sonarqube.org/latest/project-administration/narrowing-the-focus/

Reply:
"report" needs to be removed -- it's local on my system (which is why it wasn't failing) but it's not actually in the repository